helm lint (v3.10.3) fails on timescaledb-single chart

**What happened?**

Linting the timescaledb-single chart with a helm 3.10.3 fails. Failure not checked with early helm version.
https://github.com/helm/helm/issues/10149#issuecomment-980243695 concludes the error is not in helm (so far).

This PR provides a suitable fix to lint / generate a yaml valid pgbackrest.yaml file.


**Did you expect to see something different?**

No linting error.

**How to reproduce it (as minimally and precisely as possible)**:

```
+ Install helm 3.10.3.

+ helm repo add timescale https://charts.timescale.com
"timescale" already exists with the same configuration, skipping

+ helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "timescale" chart repository
Update Complete. ⎈Happy Helming!⎈

+ helm fetch timescale/timescaledb-single --untar

+ helm lint <extracted folder>
==> Linting timescaledb-single-original
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/pgbackrest.yaml: unable to parse YAML: invalid Yaml document separator: apiVersion: batch/v1
```

**Environment**

Helm: 3.10.3
values.yaml: N/A
K8s: N/A

--- a/charts/timescaledb-single/Chart.yaml
+++ b/charts/timescaledb-single/Chart.yaml
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.27.4
+version: 0.27.5
 # appVersion specifies the version of the software, which can vary wildly,
 # e.g. TimescaleDB 1.4.1 on PostgreSQL 11 or TimescaleDB 1.5.0 on PostgreSQL 12.
 # https://github.com/helm/helm/blob/master/docs/charts.md#the-appversion-field

--- a/charts/timescaledb-single/templates/pgbackrest.yaml
+++ b/charts/timescaledb-single/templates/pgbackrest.yaml
@@ -22,7 +22,7 @@ spec:
 ...
 {{- range .Values.backup.jobs }}
 ---
-{{- if semverCompare ">=1.21-0" $.Capabilities.KubeVersion.GitVersion -}}
+{{ if semverCompare ">=1.21-0" $.Capabilities.KubeVersion.GitVersion -}}
 apiVersion: batch/v1
 {{- else -}}
 apiVersion: batch/v1beta1

